### PR TITLE
Add yacht

### PR DESCRIPTION
**Pull request by BNAndras**
For #48in24.

### CODE REVIEW

File: exercises/practice/yacht/wren_modules/wren-testie
Comment (glennj) [inline exercises/practice/yacht/wren_modules/wren-testie:1]: This does not appear in other exercises.

**Review by glennj — CHANGES_REQUESTED** [review]

File: exercises/practice/yacht/wren_modules/wren-testie
Comment (BNAndras) [reply exercises/practice/yacht/wren_modules/wren-testie:1]: Oops. Should I add it to gitignore?

Comment (glennj) [reply exercises/practice/yacht/wren_modules/wren-testie:1]: It appears to be there already: https://github.com/exercism/wren/blob/main/.gitignore

Unless the leading slash is wrong

Comment (BNAndras) [reply exercises/practice/yacht/wren_modules/wren-testie:1]: I removed the folder from the index and the updated gitignore seems to be working

File: .gitignore [hunk 1]
```diff
@@ -3,4 +3,4 @@
 bin/configlet
 bin/configlet.exe
 .tmp
-/wren_modules
\ No newline at end of file
+wren_modules/
```
Comment (kotp) [inline .gitignore:6]:
```suggestion
wren_modules/

```

**Review by kotp — APPROVED** [review]

File: exercises/practice/yacht/.meta/proof.ci.wren [hunk 2]
```diff
@@ -0,0 +1,106 @@
+class Yacht {
+
+  static score(dice, category) {
```
Comment (glennj) [inline exercises/practice/yacht/.meta/proof.ci.wren:3]: If this was a mentoring session, I'd have something to say about the length of this method ;)

**Review by glennj — APPROVED** [review]

**Comment by glennj** [comment]
Thank you @BNAndras!

**Pull request merged**